todo_app/data/trello_items.py

```diff
@@ -0,0 +1,186 @@
+import requests
+import os
+import json
+
+from todo_app.data.Item import Item
+
+def get_trello_items():
+
+    board_id = os.getenv("TRELLO_BOARD_ID")
+    key = os.getenv("TRELLO_API_KEY")
+    token = os.getenv("TRELLO_API_TOKEN")
```

Comment:
There is a bit of repetition between your functions. This block, plus the headers dictionary and the query dictionary always contains at least a key and token.

You could reduce this repetition. E.g. write a helper function that builds a dictionary holding key and token. Then all of these functions use that helper and don't need to worry about TRELLO_API_KEY/TRELLO_API_TOKEN themselves. But they can still add extra query params if necessary.